testing/src/test/java/org/motechproject/nms/testing/it/kilkari/MctsBeneficiaryImportServiceBundleIT.java

```diff
@@ -1420,4 +1420,45 @@ public void testImportIntoStateWithTwoCircles() throws Exception {
         assertNotNull(subscriber);
         assertEquals("Rectangle", subscriber.getCircle().getName());
     }
+
+    @Test
+    public void testNoUpdateIfChildSubscriptionActiveForMother() throws Exception {
+        // import mother
+        DateTime lmp = DateTime.now().minusDays(100);
+        String lmpString = getDateString(lmp);
+        Reader reader = createMotherDataReader("21\t3\t\t\t\t\t1234567890\tShanti Ekka\t9439986187\t\t" +
+                lmpString + "\t\t\t");
+        mctsBeneficiaryImportService.importMotherData(reader);
+
+        // import child
+        DateTime dob = DateTime.now().minusDays(5);
+        String dobString = getDateString(dob);
+        reader = createChildDataReader("21\t3\t\t\t\t\t1234567891\tBaby1 of Shanti Ekka\t1234567890\t9439986187\t"
+                + dobString + "\t");
+        mctsBeneficiaryImportService.importChildData(reader);
+
+        //import same mother again
+        DateTime lmp1 = DateTime.now().minusDays(100);
+        String lmpString1 = getDateString(lmp1);
+        reader = createMotherDataReader("21\t3\t\t\t\t\t1234567890\tShanti Ekkat\t9439986187\t\t" +
+                lmpString1 + "\t\t\t");
+        mctsBeneficiaryImportService.importMotherData(reader);
+
+        TransactionStatus status = transactionManager.getTransaction(new DefaultTransactionDefinition());
+        Subscriber subscriber = subscriberDataService.findByNumber(9439986187L);
+        Set<Subscription> subscriptions = subscriber.getAllSubscriptions();
+
+        Subscription childSubscription = subscriptionService
+                .getActiveSubscription(subscriber, SubscriptionPackType.CHILD);
+        Subscription pregnancySubscription = subscriptionService
+                .getActiveSubscription(subscriber, SubscriptionPackType.PREGNANCY);
+
+        //the mother subscription should still be DEACTIVATED as child is ACTIVE
+        assertEquals(2, subscriptions.size());
```

Comment:
Maybe I'm confused, but why would there be 2 subscriptions on the number? Mother first, then child gets activated along with the disabling with the mother. Then you import the mother again and wouldn't create a subscription for it. 

At the end of it, you will only have 1 child subscription on the number, right? Maybe you also want to check that out of the 2 subscriptions, only the child is active?